client/actions/editActions.js

```diff
@@ -147,7 +151,7 @@ export const onChangeArticle = (key, value) => {
     }))
 
     if (!article.published) {
-      dispatch(saveArticle())
+      debouncedSaveDispatch(dispatch)
     }
```

Comment:
ooo nice this makes more sense, had been messing with debounce in the backbone 'sync' callback and was not a functional choice.